bin/npm

```diff
@@ -1,14 +1,13 @@
-#! /bin/bash
+#! /usr/bin/env bash
 
 # DOC: Run npm in the dev environment's docker container.
 
-pushd $(git rev-parse --show-toplevel)
+source bin/lib.sh
 
 # allocate a tty for better test output even though not strictly needed.
 docker run -it --rm \
   -v /var/run/docker.sock:/var/run/docker.sock \
   -v $(pwd)/universal-application-tool-0.0.1:/usr/src/universal-application-tool-0.0.1 \
   --entrypoint npm \
-  civiform-dev $@
-
-popd
+  civiform-dev \
+  $@
```

Comment:
this line wrap seems odd

Reply:
In a previous discussion with @shanemc-goog we talked about continuations in bash and iirc arrived as something along the lines of one arg or flag-arg pair per line for continuations. This is an attempt at that. I agree it looks a little funny? Open to suggestions

Reply:
mmm not especially i think i would leave it all on one line but im not super pedantic so this works for me